linked audiences use cases mini edit (#6587)
segmentio/segment-docs: src/engage/audiences/linked-audiences-use-cases.md

    - '/unify/linked-profiles/linked-audiences-use-cases'
---

> [Linked Audiences](/docs/engage/audiences/linked-audiences/) is a technical marketer’s go-to tool for creating data-driven audience segments on top of relational data in their warehouse. Built on top of our Segment [Data Graph](/docs/unify/linked-profiles/data-graph/) technology - the audience builder allows you to craft flexible segments for more targeted marketing efforts. 

Reviewer: ```suggestion
[Linked Audiences](/docs/engage/audiences/linked-audiences/) is a technical marketer’s go-to tool for creating data-driven audience segments on top of relational data in their warehouse. Built on top of Segment's [Data Graph](/docs/unify/linked-profiles/data-graph/) technology - the audience builder allows you to craft flexible segments for more targeted marketing efforts. 
```

You gain direct access to your data, enabling you to create precise audience groups for activation in your marketing tools, without relying heavily on data teams. You can take object data (For example: products, bookings, accounts) and relate it back to the profile data that lives within your warehouse for activation across any channel supported by [Engage Foundations](/docs/engage/quickstart/).

Below are some example use cases to help you learn more about Linked Audiences.

> ## Drive Accelerated Onboarding 

Reviewer: ```suggestion
## Drive accelerated onboarding 
```

We generally try to use sentence case.

An orchestrated process simplifies the onboarding experience and empowers customers to make the most of the product, turning them from first time users into loyal subscribers. With Linked Audiences, brands can create custom campaigns based on a customer’s specific account details from the warehouse and usage history from real-time event streams. 

**Example:** 
Send a one-time onboarding email when a Business Tier account is opened, and include instructions on how to get started based on corporate policies.

> ## Boost Retention and Upsell

Reviewer: ```suggestion
## Boost retention and upsell
```

Create targeted campaigns to drive retention, upsell, and cross-sell your products. With Linked Audiences, you can choose the granularity at which you trigger campaigns, based on your campaign goals. For example, a customer could have 2 cats in their household, one is a kitten that needs flea and tick medicine, while the other is a geriatric cat who needs regular checkups. With Linked Audiences, your customers can be part of a personalized pet parent journey related to each. 

**Example:** 
Send an upsell email to customers who own cats, and are also part of the Platinum membership tier audience previously created. 

## Personalized Transaction Send

> Including personalized product information in transactional emails improves open rates over generic marketing messages. Linked Audiences allows you to set up campaign triggers based on the value of an entity (account, course, pets, etc) and make it easy to enrich the message with any personalized content from the warehouse. 

Reviewer: ```suggestion
Including personalized product information in transactional emails improves open rates over generic marketing messages. Linked Audiences allows you to set up campaign triggers based on the value of an entity (account, course, pets, and so on) and make it easy to enrich the message with any personalized content from the warehouse. 
```

Latin.

**Example:** 
- Send a one-time Past Due email to customers whose accounts have just become overdue. Include the past due amount.

> - Send a weekly Course Completion Reminder email to any customer who hasn’t finished their Segment University course with details on course progress

Reviewer: ```suggestion
- Send a weekly Course Completion Reminder email to any customer who hasn’t finished their Segment University course with details on course progress
```

I feel very seen by this...
